test/georeferencing_t.cpp

```diff
@@ -342,9 +342,10 @@ void GeoreferencingTest::testGrivationAffected()
 	// Change CRS from UTM 32 to EPSG:3857.
 	// Convergence of the latter is 0, so the change in grivation should equal
 	// the present convergence.
-	georef.setProjectedCRS(QString::fromLatin1(QTest::currentDataTag()), epsg3857_spec);
+	auto const declination_setting = georef.getDeclinationSetting();
+	georef.setProjectedCRS(QString::fromLatin1(QTest::currentDataTag()), epsg3857_spec, std::vector<QString>(), false);
 	georef.setGeographicRefPoint(latlon, true, false); // Fix up scale factor, not grivation.
-	georef.updateGrivation(); // Fix up grivation.
```

Comment:
As to my understanding, this was the last use of `updateGrivation()`. Could this patch also contain the removal of the function from the code?

Reply:
That removal is in the next commit 811e4615, which removes `initDeclination()` as well as `updateGrivation()`. I like removing both together, as the two methods are twins. Would you squash the two patches?

Reply:
Thanks for the confirmation. That is what I had in mind. The method removal is logically coupled with the removal of its uses.

Reply:
I will perform the squash.